docs/concepts.rst

```diff
@@ -155,7 +283,9 @@ operators: :class:`~airflow.operators.docker_operator.DockerOperator`,
 
 Operators are only loaded by Airflow if they are assigned to a DAG.
 
-See :doc:`howto/operator/index` for how to use Airflow operators.
```

Comment:
This link only shows a subset of operators, making the documentation difficult to navigate

Reply:
👍